tests/dataset/test_aoi.py

```diff
@@ -108,7 +108,10 @@ def test_stac_url_input(self):
         assert aoi.download_data is True
         assert Path("data/SpaceNet_AOI_2_Las_Vegas-056155973080_01_P001-WV03-R.tif").is_file()
         aoi.close_raster()
-        os.remove("data/SpaceNet_AOI_2_Las_Vegas-056155973080_01_P001-WV03-R.tif")
+        try:
+            os.remove("data/SpaceNet_AOI_2_Las_Vegas-056155973080_01_P001-WV03-R.tif")
+        except PermissionError:
+            pass
```

Comment:
Fix for Windows

Reply:
Is there a test for the writing a multiband raster from VRT module?

Reply:
Yes, with this test: https://github.com/NRCan/geo-deep-learning/blob/develop/tests/dataset/test_aoi.py#L258